### PR TITLE
Update release branch to 1.11.0-rc1

### PR DESCRIPTION
**Pull request by adamwalach**
**Description**

Changes proposed in this pull request:

- adjusted Kyma repo to release 1.11.0-rc1
- cherry-picked https://github.com/kyma-project/kyma/pull/7712

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/7707

### CODE REVIEW

File: installation/resources/installer.yaml [hunk 1]
```diff
@@ -196,7 +196,7 @@ spec:
             readOnly: true
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:408cb6a6
+        image: eu.gcr.io/kyma-project/develop/installer:1.11.0-rc1
```
Comment (strekm) [inline installation/resources/installer.yaml]: it should be kyma-installer and please check folder cause i think it's not develop anymore

File: tools/kyma-installer/kyma.Dockerfile [hunk 2]
```diff
@@ -1,7 +1,4 @@
-ARG INSTALLER_VERSION="c74d3125"
-ARG INSTALLER_DIR=eu.gcr.io/kyma-project
-FROM $INSTALLER_DIR/kyma-operator:$INSTALLER_VERSION
-
+FROM eu.gcr.io/kyma-project/kyma-operator:master-c74d3125
```
Comment (strekm) [inline tools/kyma-installer/kyma.Dockerfile]: shouldn't it be 1.11.0-rc1 as well?

File: installation/resources/installer.yaml (hunk 1 above)
Comment (adamwalach) [reply installation/resources/installer.yaml]: you are right, thanks!

File: tools/kyma-installer/kyma.Dockerfile (hunk 2 above)
Comment (adamwalach) [reply tools/kyma-installer/kyma.Dockerfile]: This line is modified according to the documentation.
There is no ```eu.gcr.io/kyma-project/kyma-operator:1.11.0-rc1``` in the registry.

**Pull request merged**